### ISSUE TITLE
Missing changelogs

### ISSUE THREAD

**Issue by JorisDebonnet** [tractorcow-farm/silverstripe-fluent#537]
There are a few issues with the Changelogs:

- 4.3.0 and 4.4.0 were recently released, in quick succession (which in itself already confuses me a bit). But I don't really know what the difference between them are because the release tags do not include any information. All recent releases before 4.3.0 were tagged by robbieaverill and included much more information. Would be great for the quality of this module if that could be done again moving forward :)
- `CHANGELOG.md` is hopelessly out of date. I suggest either updating it or removing the link to this file from `readme.md`.

So in short, it would be great for users if it could be made clearer what the releases are about. Thanks!

**Comment by ScopeyNZ** [comment]
Here's a git command I use to generate changelogs:

```
git log --oneline --pretty=format:"* %s (%an) - %h" --no-merges
```

I alias that to `gcl` on my machine. Running `gcl 4.2.1..4.3.0` gives:

```
* Update SilverStripe test versions in Travis (Robbie Averill) - 13d0cc1
* Fix import CheckboxSetField (Ian Patel) - d147111
* Replace filter grid with checkbox set (Ian Patel) - 6040646
* add .upgrade.yml (wernerkrauss) - cf0c2f7
* FIX Fluent now respects existing base URL prefixes in URL segment fields when no fluent domain exists (Robbie Averill) - 83f9fb9
* Add PHP 7.3 and SilverStripe 4.3/4.4 to Travis builds (Robbie Averill) - 4d016e0
* Update translations (Robbie Averill) - 997a1a9
* Use the locale's URLSegment as default badge label (JorisDebonnet) - 73339bf
* Clearer Versioned example (Ingo Schommer) - 2c94814
* fix index when fluent is in use and there is no fallback locale (Aljoša Balažic) - 4b0eacf
* move opening brace to a new line (AljosaB) - 63e4252
* add alternate links to metatags (Aljoša Balažic) - e91e86a
* Bump alias for master to 4.3.x-dev (Robbie Averill) - 7de2a23
```

I agree about removing the changelog file. It's pretty annoying to keep that up to date. I'll wait for further discussion before I delete it though 🙂

**Comment by tractorcow** [comment]
I have been a bit lazy sorry; I'll retroactively add a changelog generator for the missing releases.

**Comment by robbieaverill** [comment]
I've updated the release notes on the GitHub releases and deleted the changelog file from all branches.
